plugins/velocity/src/main/java/org/apache/struts2/views/velocity/VelocityManager.java

```diff
@@ -161,32 +164,24 @@ public Context createContext(ValueStack stack, HttpServletRequest req, HttpServl
      * perform any initialization of the contexts.  All that must be done in the
      * context itself.
      *
-     * @param servletRequest the servlet request object
+     * @param servletRequest  the servlet request object
      * @param servletResponse the servlet response object
-     * @param extraContext map with extra context
+     * @param extraContext    map with extra context
```

Comment:
Was this an unintentional whitespace change ?

Reply:
The same case with auto-format